src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -116,6 +116,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('account_name', acct_name_type, options_list=['--name', '-n'], completer=None)
         c.argument('kind', help='Indicates the type of storage account.',
                    arg_type=get_enum_type(t_kind, default='storage'))
+        c.argument('https_only', arg_type=get_three_state_flag(), min_api='2019-04-01',
+                   help='Allow https traffic only to storage service if set to true. The default value is true.')
+        c.argument('https_only', arg_type=get_three_state_flag(), max_api='2018-11-01',
```

Comment:
why do we need two lines?

Reply:
A long story... Service team change the default values for different API version. So I have to split them and give the corresponding help message..

Reply:
Same help twice. You can assign it to a variable and reference it.
Same arg_type, help, use CLIArgumentType?

Reply:
It is not the same. They are different.

Reply:
'Allow https traffic only to storage service if set to true. The default value is true.' twice.
Minor issue🙂

Reply:
the second sentence is "'Allow https traffic only to storage service if set to true. The default value is **false**"

Reply:
@qwordy

Reply:
Oh! I'll go to ophthalmology department tomorrow.

Reply:
> ophthalmology

well. You don't need. It's really hard to find.....